sections/06_discussion.md

```diff
@@ -2,57 +2,78 @@
 
 Despite the disparate types of data and scientific goals in the learning tasks
 covered above, several challenges are broadly important for deep learning in the
-biomedical domain.  Here we examine these factors that may impede further
+biomedical domain. Here we examine these factors that may impede further
 progress, ask what steps have already been taken to overcome them, and suggest
 future research directions.
 
-### Evaluation
-
-There are unique challenges to evaluating deep learning predictions in the
-biomedical domain. We focus on TF binding prediction as a representative task to
-illustrate some of these issues. The human genome has 3 billion base pairs, and
-only a small fraction of them are implicated in specific biochemical activities.
-As a result, classification of genomic regions based on their biochemical
-activity results in highly imbalanced classification.  Class imbalance also
-arises in other problems we review, such as virtual screening for drug
-discovery. What are appropriate evaluation metrics that account for the label
-imbalance? The classification labels are formulated based on continuous value
-experimental signals. Practitioners must determine an appropriate procedure for
-formulating binary classification labels based on these signals. In addition,
-the experimental signals are only partially reproducible across experimental
-replicates.  An appropriate upper bound for classification performance must
-account for the experimental reproducibility.
+### General Modeling and Evaluation Considerations
+
+There are challenges with fitting and evaluating deep learning predictions in
+the biomedical domain. To some extent, these difficulties are common to all
+machine learning methods applied to biomedical data. However, the utilization of
+large datasets with high dimensional feature sets for deep learning can
+exacerbate these issues.
+
+Many of the variations on deep learning models described in this review reflect
+design choices informed by foundational machine learning tradeoffs -
+specifically, tuning the flexibility of a model to maximize predictivity for a
```

Comment:
I think this is a pretty interesting topic point. My current take is the "traditional" perspectives seem to come up most often in the small data regime (by deep learning standards) which is fairly relevant to biological datasets. An extreme (but pretty silly and not-so-informative) example is the exchange between Leek and Beam http://beamandrew.github.io/deeplearning/2017/06/04/deep_learning_works.html). 

On the other hand, it's true that deep learning models and their properties challenges some of our basic intuitions around model regularization tradeoffs.

Two papers that come to mind are Zhang et al's "Understanding deep learning requires rethinking generalization" and Lin's "Why does deep and cheap learning work so well?". However it's clear there's more theoretical to be done for the field to obtain some clarity, let alone explain this to a non-technical audience well. I didn't elaborate on this initially because I thought it might be too technical, but perhaps it warrants a mention given this is a point of distinction.